java-library/src/main/java/com/microsoft/azure/functions/sql/annotation/SQLInput.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions.sql.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.ElementType;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+@CustomBinding(direction = "in", name = "sql", type = "sql")
+public @interface SQLInput {
+    /**
+     * Query string or name of stored procedure to be run.
+     */
+    String commandText() default "";
+
+    /**
+     * Text or Stored Procedure.
+     */
+    String commandType() default "";
```

Comment:
we're defaulting to Text on CSharp, shouldn't we do the same here instead of leaving it empty?

Reply:
I tried to use Text as default but when I ran a Java function without the commandType specified, it threw an error:
`Microsoft.Azure.WebJobs.Extensions.Sql: The type of the SQL attribute for an input binding must be either CommandType.Text for a direct SQL query, or CommandType.StoredProcedure for a stored procedure.`

I'll leave it empty for now but can certainly revisit this.

Reply:
Please file an issue so we can track that